### PR TITLE
bump gcp deps

### PR DESCRIPTION
**Pull request by danielnorberg**
Latest datastore + bigtable libs + required deps

### CODE REVIEW

**Comment by danielnorberg** [comment]
@bergman PTAL

File: styx-api-service/src/test/java/com/spotify/styx/api/BackfillResourceTest.java [hunk 1]
```diff
@@ -140,7 +141,11 @@ public static void setUpClass() throws Exception {
   @AfterClass
   public static void tearDownClass() throws Exception {
     if (localDatastore != null) {
-      localDatastore.stop();
+      try {
+        localDatastore.stop(org.threeten.bp.Duration.ofSeconds(30));
```
Comment (bergman) [inline styx-api-service/src/test/java/com/spotify/styx/api/BackfillResourceTest.java]: This looks like a weird place to find joda-time.

File: styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java [hunk 2]
```diff
@@ -22,10 +22,10 @@
 
 import static com.spotify.styx.serialization.Json.OBJECT_MAPPER;
 
+import com.google.cloud.Timestamp;
```
Comment (bergman) [inline styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java]: is Timestamp the new DateTime? Are they compatible?

File: styx-api-service/src/test/java/com/spotify/styx/api/BackfillResourceTest.java (hunk 1 above)
Comment (danielnorberg) [inline styx-api-service/src/test/java/com/spotify/styx/api/BackfillResourceTest.java]: it's not joda-time but a library with a few backports of the new java 8 time api classes. I guess the datastore library uses it to not require java 8.

File: styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java (hunk 2 above)
Comment (danielnorberg) [inline styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java]: Afaict yes http://googlecloudplatform.github.io/google-cloud-java/0.17.1/apidocs/com/google/cloud/datastore/BaseEntity.html#getTimestamp-java.lang.String-

**Comment by danielnorberg** [comment]
Ran this against real datastore & bigtable, was able to successfully trigger and run a workflow etc.

**Pull request merged**